src/main/java/clickup/ui/entities/Context.java

```diff
@@ -17,4 +17,28 @@
  * @version 1.0
  */
 public class Context {
+   private Space space;
+
+    /**
+     * Constructor.
+     */
+    public Context() {
+        this.space = new Space();
+    }
+
+    /**
+     * Method let the ger the space.
```

Comment:
Please change the documentation according to our Code Conventions and add an empty line.

Reply:
Done.